app/lib/submission_builder/ty2022/states/az/individual_return.rb

```diff
@@ -4,12 +4,15 @@ module Ty2022
     module States
       module Az
         class IndividualReturn < SubmissionBuilder::Document
-          FILING_STATUSES = {
-            single: 'Single',
-            married_filing_jointly: 'MarriedJoint',
-            married_filing_separately: 'MarriedFilingSeparateReturn',
-            head_of_household: 'HeadHousehold',
-          }.freeze
+          include DependentRelationshipTable
+
+          FILING_STATUS_OPTIONS = {
+            :married_filing_jointly => 'MarriedJoint',
+            :head_of_household => 'HeadHousehold',
+            :married_filing_separately => 'MarriedFilingSeparateReturn',
+            :single => "Single"
+          }
```

Comment:
This was already in the file, much lower down. I moved it up next to the other constants.